citi-onboarding/Recorda-one-page
# feat(feature/CarouselCRUD): Finalized carouselCRUD

#What I did:

- Created the Carousel model
- Created the CarouselController
- Created the CRUD routes to the CarouselController
- Deleted the User model, UserController and the CRUD routes to the UserController

#How to test

1. git checkout main
2. git pull
3. git checkout feature/CarouselCRUD
4. git pull
5. Open terminal in the "server" folder
6. Run "npm instal" or "yarn install"
7. Run "docker compose up" or "docker-compose up
8. Open Postbird and fill in the folder ".env" information
9. Open Insomnia and connect with localhost
10. Test with "get", "put", "post" and "delete" at Insomnia
11. Check if it's all working on Postbird
